adapters/rubicon/rubicon.go

```diff
@@ -945,6 +941,58 @@ func updateUserExtWithIabAttribute(userExtRP *rubiconUserExt, data []openrtb2.Da
 	return nil
 }
 
+func updateSiteExtWithIabAttribute(siteExtRP *rubiconSiteExt, data []openrtb2.Data) error {
+	siteSegtaxes := []int{1, 2}
+	var segmentIdsToCopy = getSegmentIdsToCopy(data, siteSegtaxes)
+
+	siteExtRPTarget := make(map[string]interface{})
+
+	if siteExtRP.RP.Target != nil {
```

Comment:
Would this condition ever be true? Because based on your code above on L857, you never set the `Target` field

Reply:
Nice catch)